### PR TITLE
Migrate "easy event source" tutorial to Cloudevents javascript sdk v2

### PR DESCRIPTION
**Pull request by aliok**
## Proposed Changes 

- Migrate the tutorial to use Javascript SDK v2
- Needs Javascript SDK 2.0.1 (not released) because of https://github.com/cloudevents/sdk-javascript/issues/191, thus references a commit

### CODE REVIEW

File: docs/eventing/samples/writing-event-source-easy-way/package.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "cloudevents-sdk": "^1.0.0"
+    "cloudevents-sdk": "git://github.com/cloudevents/sdk-javascript.git#0710166ce9397f402b835fae745923d11357d15e"
```
Comment (aliok) [inline docs/eventing/samples/writing-event-source-easy-way/package.json:12]: Gonna change this once the Javascript SDK 2.0.1 is released.

**Comment by aliok** [comment]
Let's merge this PR that way and I will change it to use proper CloudEvents Javascript SDK release later

**Comment by aliok** [comment]
/unhold
cc @slinkydeveloper @evankanderson 

@lance --> feel free to take a look if you're interested

File: docs/eventing/samples/writing-event-source-easy-way/README.md [hunk 2]
```diff
@@ -19,9 +19,12 @@ Code for this tutorial is available [here](https://github.com/knative/docs/tree/
 Create the project and add the dependencies:
 ```bash
 npm init
-npm install cloudevents-sdk --save
+npm install cloudevents-sdk@2.0.1 --save
 ```
 
+Please note that because of a [bug](https://github.com/cloudevents/sdk-javascript/issues/191), you will
+need at least `2.0.1` version of the Javascript SDK.
```
Comment (mattmoor) [inline docs/eventing/samples/writing-event-source-easy-way/README.md:26]: Remove trailing whitespace:
```suggestion
need at least `2.0.1` version of the Javascript SDK.
```

File: docs/eventing/samples/writing-event-source-easy-way/package.json (hunk 1 above)
Comment (aliok) [reply docs/eventing/samples/writing-event-source-easy-way/package.json]: Changed now

**Comment by n3wscott** [comment]
so nice.

/lgtm
/approve

**Pull request merged**